src/db/models/user.model.js

```diff
@@ -129,11 +142,90 @@ module.exports = (sequelize, DataTypes) => {
         });
     };
 
+    User.verifyAccountToken = function verifyAccountToken(email, expTime) {
+        // This expects and email, and an expiration time window for storing a
+        // messaging protocol `messagingProtocol` token, auth expiration, and
+        // provider for a user. Users will be asked to validate this token against
```

Comment:
This function always performs a create-or-update, correct? If so, can we name it `createOrUpdateVerificationToken` or `upsertVerificationToken`? Note: "Upsert" generally refers to inserting a new row in a table, but IMO it's ok to use it here to describe adding a value to a NULL field in an existing row.

I don't understand what the first sentence here is saying.

Reply:
Done.